src/Testing/src/PublicAPI.Unshipped.txt

```diff
@@ -1,2 +1,5 @@
 #nullable enable
 ~Microsoft.AspNetCore.Testing.QuarantinedTestAttribute.QuarantinedTestAttribute(string reason) -> void
+Microsoft.AspNetCore.Testing.HelixConstants
+~const Microsoft.AspNetCore.Testing.HelixConstants.Windows10Arm64 = "Windows.10.Arm64v8.Open;" -> string
```

Comment:
Do we ship this package?

Reply:
We better not.

Reply:
No we didn't, we should just remove the API baseline for this package.